build/task-link.x

```diff
@@ -94,6 +94,12 @@ SECTIONS
     KEEP(*(.task_slot_table));
   }
 
+  /* ## .idolatry */
+  .idolatry (INFO) : {
+    . = .;
```

Comment:
...what's this line for?

Reply:
This is to assure that the `.idolatry` section isn't chucked.